src/js/network-vis.js

```diff
@@ -1,9 +1,12 @@
 import { Network } from 'vis-network/peer/esm/vis-network';
```

Comment:
please rename this file to network_vis to stay consistent with other files.

Reply:
This is fine, hyphen is the norm for file naming in javascript. Will update older files.